test/plugin-e2e-specs.js

```diff
@@ -84,7 +89,7 @@ describe('FakePlugin', function () {
     });
     it('should not handle commands if plugin is not activated', async function () {
       const driver = wd.promiseChainRemote(TEST_SERVER);
-      const [sessionId] = await driver.init(caps);
+      const [sessionId] = await driver.init(w3cCaps);
```

Comment:
ah, here should be replaced to webdriverio or another simple request (probably this is better) for W3C since current client is wd.